Code/Framework/AzCore/AzCore/Settings/SettingsRegistryMergeUtils.cpp

```diff
@@ -245,6 +253,36 @@ namespace AZ::Internal
 
 namespace AZ::SettingsRegistryMergeUtils
 {
+    //! The algorithm that is used to find the *directory* containing the o3de_manifest.json
+    //! 1. It first checks the "{BootstrapSettingsRootKey}/o3de_manifest_path"
+    //! 2. If that is not set it defaults to the user's home directory with the .o3de folder appended to it
+    //!    equal to "~/.o3de"
```

Comment:
suggestion: Is it worth also including `C:\Users\<user>\.o3de` for Windows users too?

Reply:
What the home directory is located at is actually explained in the [SettingsRegistryMergeUtils](https://github.com/o3de/o3de/blob/1bd21542d36082fc4434ffc8e9c05d26c55c8b90/Code/Framework/AzCore/AzCore/Settings/SettingsRegistryMergeUtils.h#L239-L240) header file.

Reply:
Okay nice, referring to the `%USERPROFILE%` env variable is a good shout